githubapp/middleware_logging.go

```diff
@@ -117,6 +139,14 @@ func LogResponseBody(patterns ...string) ClientLoggingOption {
 	}
 }
 
+// SetRateLimitInformation defines which rate limit information like
+// the number of requests remaining in the current rate limit window is getting logged.
+func SetRateLimitInformation(options *RateLimitLoggingOption) ClientLoggingOption {
```

Comment:
Let's call this `LogRateLimitInformation` to align with the other `ClientLoggingOption` functions

Reply:
Fixed in https://github.com/palantir/go-githubapp/pull/413/commits/74b6344966e743226b495271824618974cd770ee